homeassistant/components/almond/strings.json

```
@@ -8,7 +8,7 @@
       }
     },
     "abort": {
-      "already_setup": "You can only configure one Almond account.",
+      "already_setup": "[%key:common::config_flow::abort::single_instance_allowed%].",
```

Please replace the key as well and adjust config_flow.py and the tests accordingly to be consistent with other integrations.
```suggestion
      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%].",
```